components/integrations/components/installed_incoming_webhooks/installed_incoming_webhooks.jsx

```diff
@@ -70,7 +75,7 @@ export default class InstalledIncomingWebhooks extends React.PureComponent {
 
     componentDidMount() {
         if (window.mm_config.EnableIncomingWebhooks === 'true') {
-            this.props.actions.getIncomingHooks().then(
+            this.props.actions.getIncomingHooks(this.props.teamId).then(
```

Comment:
You'll also want to increase the page size here to 10000 to match [the old behaviour](https://github.com/mattermost/mattermost-webapp/blob/2dc41955086da015fc78f54c61a71888e96a815f/actions/integration_actions.jsx#L23). Eventually, we'll want to have this list properly split into pages, but that can be done later. For now, it's important to show all the webhooks for teams that have a lot

Reply:
I can amend this commit with the changes. Should I add `10000` as a constant or use an existing one like `MAX_POST_VISIBILITY: 1000000,`